src/main/webapp/requestResendLink.jsp

```diff
@@ -0,0 +1,23 @@
+<%@ page pageEncoding="UTF-8" %>
+<%@ taglib tagdir="/WEB-INF/tags" prefix="t" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<c:set var="jsIncludes">
+  <script src="https://www.google.com/recaptcha/api.js" async defer></script>
+  <script type="text/javascript" src="/js/requestResendLink.js"></script>
+</c:set>
+<t:statusMessage statusMessagesToUser="${data.statusMessage}"/>
+<t:staticPage jsIncludes="${jsIncludes}">
+  <link type="text/css" href="/stylesheets/teammatesCommon.css" rel="stylesheet">
+  <h1 class="color-orange">
+    Request for Resend of Feedback Links
+  </h1>
+  <p id="message">Please enter your email address, an email containing links to all the feedback sessions that you participated in the recent one month will be resend to you.</p>
+  <form id="requestForm" action="/requestResendLinkSuccess.jsp" name="requestForm" method="POST">
+    <input id="email" class="width-200-px" name="studentemail" type="text" placeholder="Enter your email address">
```

Comment:
What I meant by email validation earlier is front-end built-in validation, i.e. just `type="email"`.

Reply:
To ensure that the status messages are synchronised, I will just do the backend validation